docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md

```diff
@@ -104,114 +104,13 @@ From Keycloak versions 21+, the default JDBC driver can be overwritten, allowing
 
 The following example uses the mentioned `aws-advanced-jdbc-wrapper`. Additionally, see the [Keycloak documentation](https://www.keycloak.org/server/db#_overriding_the_default_jdbc_driver) on overwriting the default JDBC driver.
 
```

Comment:
You can remove this line since we don't have an example anymore, nor does the customer have to figure out how to overwrite the default JDVC driver.